src/App.tsx

```diff
@@ -26,70 +26,128 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
-export function getReorderedGoods(
-  goods: string[],
-  { sortType, isReversed }: ReorderOptions,
-) {
-  // To avoid the original array mutation
-  const visibleGoods = [...goods];
+export class App extends PureComponent<{}, ReorderOptions> {
+  state: ReorderOptions = {
+    isReversed: false,
+    sortType: SortType.NONE,
+  };
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  handleSortAlphabetically = () => {
+    this.setState({
+      sortType: SortType.ALPHABET,
+    });
+  };
 
-  return visibleGoods;
-}
+  handleSortByLength = () => {
+    this.setState({
+      sortType: SortType.LENGTH,
+    });
+  };
```

Comment:
no need to add `handle` to each function name, if it performs only one action. this does not give any additional info